Marlin/Configuration.h

```diff
@@ -1170,6 +1170,12 @@
  * Filament Runout Sensors
  * Mechanical or opto endstops are used to check for the presence of filament.
  *
+ * IMPORTANT: Runout will only trigger if Marlin is aware that a print job is running.
+ * Marlin knows a print job is running when:
+ *  1. Running a print job from media started with M24.
+ *  2. The Print Job Timer has been started with M75.
+ *  3. The heaters were turned on and PRINTJOB_TIMER_AUTOSTART is enabled.
```

Comment:
You need to send M109/M191 to start the timer. Pre heat will not start the timer.

Reply:
And those issues were exactly about users doing preheat and removing M109/M119 from gcode, using serial.

Reply:
We don't need this paragraph to be too detailed since this aspect of Marlin is documented elsewhere. I'll see if I can find a link…

Reply:
I agree. I just think that if we listed already 2 gcode, why not tone more? :-)

Reply:
This sort of thing inspires me to push on with the Config parser in Auto Build Marlin so all this stuff can be used to generate custom editing, configurator form fields, and documentation pages. Seeing our explanations in the context of user documentation and tool-tips can help to drive its development.